src/databricks/labs/blueprint/installation.py

```diff
@@ -40,6 +40,8 @@
 
 __all__ = ["Installation", "MockInstallation", "IllegalState", "NotInstalled", "SerdeError"]
 
+FILE_SIZE_LIMIT: int = 1024 * 1024 * 10
```

Comment:
Type hint lookgs redudant

```suggestion
FILE_SIZE_LIMIT = 1024 * 1024 * 10
```

Reply:
Irrespective of whether it's redundant or not, it's incorrect: it should be `ClassVar[int]`.